pom.xml

```diff
@@ -89,6 +95,21 @@
             <artifactId>spring-context</artifactId>
             <version>${spring-core.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+            <version>${spring-core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>${spring-core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-core</artifactId>
+            <version>${spring-core.version}.RELEASE</version>
```

Comment:
should consider define separate variable for spring-security-core.version with 5.3.4.RELEASE as value.

Reply:
Fixed. Also changed spring-security-core version to the latest supported